winpr/include/winpr/crt.h

```diff
@@ -146,19 +148,7 @@ extern "C"
 {
 #endif
 
-	WINPR_API void* _aligned_malloc(size_t size, size_t alignment);
-	WINPR_API void* _aligned_realloc(void* memblock, size_t size, size_t alignment);
-	WINPR_API void* _aligned_recalloc(void* memblock, size_t num, size_t size, size_t alignment);
-
-	WINPR_API void* _aligned_offset_malloc(size_t size, size_t alignment, size_t offset);
-	WINPR_API void* _aligned_offset_realloc(void* memblock, size_t size, size_t alignment,
-	                                        size_t offset);
-	WINPR_API void* _aligned_offset_recalloc(void* memblock, size_t num, size_t size,
-	                                         size_t alignment, size_t offset);
-
-	WINPR_API size_t _aligned_msize(void* memblock, size_t alignment, size_t offset);
-
-	WINPR_API void _aligned_free(void* memblock);
+#ifndef _WIN32
```

Comment:
This looks wrong, should be before the `#ifdef __cplusplus` above, shouldn´t it?

Reply:
good catch.